dwh/indicator_lib.py

```diff
@@ -65,6 +65,22 @@ def _agg_buckets(birth_date: str, gender: str, end_date: datetime) -> List[str]:
   return [age_band + '_' + gender, 'ALL-AGES_' + gender,
           age_band + '_ALL-GENDERS', 'ALL-AGES_ALL-GENDERS']
 
+def _gen_counts_and_ratio(temp_df: pandas.DataFrame,  end_date: datetime,
+                         indicator_name: str,) -> pandas.DataFrame:
+  """Generates aggregated dataframe when supplied with patient-level df"""
+  temp_df['buckets'] = temp_df.apply(
+    lambda x: _agg_buckets(x.birthDate, x.gender, end_date), axis=1)
+  temp_df_exp = temp_df.explode('buckets')
+  temp_df_exp = temp_df_exp.groupby([indicator_name, 'buckets'], as_index=False) \
```

Comment:
As a general note, I think using backslash is discouraged to break [long lines](https://google.github.io/styleguide/pyguide.html#32-line-length).